### DISCUSSION TITLE
Move settings to a modal window

### DISCUSSION THREAD

**Discussion by J-Cake** [keepassxreboot/keepassxc#10752] [Ideas]
Hi, I'm hoping to figure why the settings are implemented the way they are? I feel like it's more intuitive tohave a pop-out modal for preferences instead of filling the main window. 
If this were to be changed, how much extra work would be involved in doing so?

**Comment by droidmonkey** [comment]
Why would a modal window be better? 

We won't change if, it's been this way since the program was created by the original owners

**Reply by J-Cake** [reply]
Oh I'm not asking for a change, it just _seems more intuitive_ to me as someone who uses the software regularly. I'm hoping for insight - something which may help see the reasoning behind it more clearly

**Reply by droidmonkey** [reply]
It's just the way it is, we prefer to not use modal windows.
